engine/src/main/java/org/terasology/rendering/nui/widgets/UISlider.java

```diff
@@ -16,26 +16,24 @@
 package org.terasology.rendering.nui.widgets;
 
 import com.google.common.base.Function;
+import org.terasology.input.Keyboard;
 import org.terasology.input.MouseInput;
+import org.terasology.input.events.MouseWheelEvent;
 import org.terasology.math.geom.Rect2i;
 import org.terasology.math.TeraMath;
 import org.terasology.math.geom.Vector2i;
-import org.terasology.rendering.nui.BaseInteractionListener;
-import org.terasology.rendering.nui.Canvas;
-import org.terasology.rendering.nui.CoreWidget;
-import org.terasology.rendering.nui.InteractionListener;
-import org.terasology.rendering.nui.LayoutConfig;
-import org.terasology.rendering.nui.SubRegion;
+import org.terasology.rendering.nui.*;
```

Comment:
we don't use .* operator when importing library. you can configure intellj to expand everything. 

https://odoepner.wordpress.com/2015/08/12/configure-intellij-to-use-default-eclipse-java-import-layout/